day_0/README.md

```diff
@@ -1,228 +1,144 @@
+![Spotlight](./images/spotlight.png)
 
-Here's what we're going to go over:
+Go ahead and start typing `Terminal`, and you should see your computer attempt to locate the file or program you're trying to find. If you have a lot of other programs or files with `term` in the name, you might have to type the full word, but if not, you should be able to type a letters until Spotlight correctly guesses that you're looking for the program `Terminal`. Once you see that, hit return and your terminal should open.
 
-*   Setting up Terminal Access for Atom
-*   Setting up GitHub SSH Keys
-*   Installing Rbenv - a Ruby Version Manager. This allow us to install other versions as needed
-*   Installing Ruby Versions using Rbenv
-*   Forking and Cloning the Prework Repository
+It will likely look something like the image below.
 
+![Terminal](./images/terminal.png)
 
-### Setting Up Terminal Access for Atom
+Now, type `ls` and hit return. You should see a list of directories that are in your home directory on your computer. You should be able to confirm this by opening the finder (go ahead and hit `command-space` to activate Spotlight and start typing `finder`) and navigating to that same home directory. Notice in the image below that my terminal window lists `Applications`, and `Desktop`, and `Documents`, etc. while my finder shows those same directories.
 
-Install the shell commands for Atom. Open Atom, drop down the `Atom` menu in the top left corner of your screen, and click on `Install Shell Commands`. Atom should now be enabled from your command line.
+![Finder](./images/finder.png)
 
-To confirm that atom is working from your command line, open a terminal with Spotlight search (`Command + Space`), type "terminal" and hit enter. Type `atom .` in your terminal. If it is setup correctly, the atom editor will automatically open. If it does not open atom and an error occurs instead, try entering this in the command line: `ln -s /Applications/Atom.app/Contents/Resources/app/atom.sh /usr/local/bin/atom` and try the first command (`atom .`) again.
+**This is key:** the terminal gives you a different way to work with your computer, but it's not completely separate. All of the files that you create and edit with your terminal, you can also view with your finder and a text editor or other program.
 
-Atom also offers a number of different options and packages that you can customize to your liking. [This](https://www.youtube.com/watch?v=WWwBQQOGllo&list=PLYzJdSdNWNqwNWlxz7bvu-lOYR0CFWQ4I) series of videos will walk you through many of them if you'd like to dive deeper.
+There are **many** commands that you can learn before you would be considered an expert at using the terminal, but you only really need a few to get started. Let's work on those below.
 
-### GitHub SSH Keys
+## Getting Around
 
-SSH keys are a more secure and convenient way of authenticating than typing in our password every time we want to interact with Github.
+### Commands
```

Comment:
We do go over all of these commands in the mod 0 curriculum. Not sure if it's super helpful to go over them again or if it would make sense to link back to the mod 0 curriculum site. Ideally we're not creating more to maintain by going over material we go over in depth in mod 0.

Reply:
Agreed 100%! This was leftover stuff from when this repo was first created; I'll clean up the duplicate stuff, link to mod0 where appropriate.

Reply:
Agreed 100%! This was leftover stuff from when this repo was first created; I'll clean up the duplicate stuff, link to mod0 where appropriate.